docs/docs/features/split-keyboards.md

```diff
@@ -0,0 +1,92 @@
+These files need to be flashed to the respective controllers of the two halves.
+
+:::tip[Updating your keymap]
+Since the keymap processing is primarily done on the central side, for keymap changes it will typically be enough to flash the controller of the central half.
+However if you make changes to [config files](../config/index.md) that should apply to all parts, you need to flash to all parts.
+Any changes in ZMK related to split keyboard features might also necessitate doing this.
+:::
+
+## Pairing for Wireless Split Keyboards
+
+Split keyboards with BLE-based split communications (i.e. all officially supported split keyboards) have an internal pairing procedure between the central and each peripheral.
+When the central has an open slot for a peripheral, it will advertise for connections (which will not be visible to non-ZMK devices).
+Then, any peripheral that has not yet bonded to a peripheral will pair to it.
+Similar to how [bluetooth profiles](bluetooth.md) are managed between the keyboard and host devices, the bonding information will be stored with the corresponding hardware addresses of the other keyboard part, on both the central and peripheral.
+
+In practice, this means that your split keyboard parts will automatically pair and work the first time they are all on at the same time.
+However, if this process somehow went wrong or you used controllers in a different split keyboard configuration before, you will need to explicitly clear the stored bond information so that the parts can pair properly.
+For this, please follow [the specified procedure](../troubleshooting/connection-issues.mdx#split-keyboard-halves-unable-to-pair) in the troubleshooting section.
+
+:::warning
+If the central keyboard part is either advertising for a pairing or waiting for disconnected peripherals, it will consume more power and drain batteries faster.
+:::
+
+## Behaviors with Locality
+
+Most ZMK [behaviors](../behaviors/index.mdx) are processed exclusively on the central part of the split keyboard since it handles the keymap state and any communication with the host devices.
+However, certain behaviors have "global" or "source" localities, where they can have an effect the peripheral parts when invoked.
+
+### Global Locality Behaviors
+
+These are behaviors that have an effect on all keyboard parts, such as changing lighting effects.
+Currently these are the following behaviors:
+
+- [RGB underglow behaviors](../behaviors/underglow.md)
+- [Backlight behaviors](../behaviors/backlight.md)
+- [Power management behaviors](../behaviors/power.md)
+- [Soft off behavior](../behaviors/soft-off.md)
+
+### Source Locality Behaviors
+
+These behaviors only affect the keyboard part that they are invoked from, given that they were invoked from a plain behavior binding (i.e. not nested inside another behavior) on a keymap layer. These behaviors include:
```

Comment:
What happens if you invoke reset from a tap dance?

Reply:
This is referring to #1494 and yeah, it would apply to tap dances as well. On second thought, since this causes issues with the global behaviors as well I might move this to a warning box that applies to both?

Reply:
I think that would be good. Maybe get rid of the "Given the were invoked" part, and move all of that into a warning box which talks about nesting (and perhaps also list the nested behaviours we have? If that ends up looking nice)

Reply:
I didn't put a full list but hopefully covered the common cases. Uncommon cases would be sensor rotate, perhaps macro (I haven't tested this one)